pinot-controller/src/main/java/org/apache/pinot/controller/api/resources/PinotSegmentRestletResource.java

```diff
@@ -901,6 +903,35 @@ public String getServerMetadata(
     return segmentsMetadata;
   }
 
+  @GET
+  @Path("segments/{tableName}/zkmetadata")
+  @Produces(MediaType.APPLICATION_JSON)
+  @ApiOperation(value = "Get the zookeeper metadata for all table segments",
+          notes = "Get the zookeeper metadata for all table segments")
+  public String getZookeeperMetadata(
+          @ApiParam(value = "Name of the table", required = true) @PathParam("tableName") String tableName,
```

Comment:
Please apply the [Pinot Style](https://docs.pinot.apache.org/developers/developers-and-contributors/code-setup#setup-ide) and reformat the changes